test/e2e/iam/project_settings_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || iam
```

Comment:
it looks like we're executing this test also for cloud manager, which we don't want to, now I'm wondering why the atlas && iam tags didn't work previously

@fmenezes is this the first test on iam we need to run for atlas only? maybe we need to update `cloud_manager_iam_e2e` or `atlas_iam_e2e` to use `E2E_TAGS: iam,atlas` or do that to CM?

Reply:
I think this should be
```suggestion
//go:build e2e || (iam && atlas)
```

iam tests run on mongocli cm/om/atlas

Reply:
nvm me I'm checking

Reply:
I think I've grasped it, this new command only works for atlas but in mongocli `iam` subcommands are for CM, OM and Atlas.

Most likely we will need to expose `Builder()` for mongocli and `AtlasCLIBuilder()` in `projects.go` for AtlasCLI and move the test to atlas generic.

Reply:
cool, that makes more sense it should not show up on the mongocli at all, thanks!